python/tank/descriptor/descriptor_cached_config.py

```diff
@@ -34,7 +34,7 @@ class CachedConfigDescriptor(ConfigDescriptor):
     interpreter files that live alongside the configuration, even tough technically these files
     are part of the core's configuration.
     """
-    def _get_config_folder(self):
+    def get_config_folder(self):
```

Comment:
The CachedConfigDescriptor and InstalledConfigDescriptor APIs are not public actually. Only the ConfigDescriptor is  actually documented in our doc. I remember that we had decided to only publicly document the base class since the functionality was the same for both. So for now I've document both behaviours in the base class's documetation.

If that becomes a problem we can always revisit de doc.

Reply:
sorry, i remember now! i got confused since it was in the `descriptor` module.